### PR TITLE
Refactor test app

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets |
| License         | MIT

it replaces #205

### CODE REVIEW

**Comment by loic425** [comment]
@pamil @lchrusciel I reworked this test app refactoring :)

File: src/Bundle/test/config/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    $_SERVER += $env;
+    $_ENV += $env;
+} elseif (!class_exists(Dotenv::class)) {
+    throw new RuntimeException('Please run "composer require symfony/dotenv" to load the ".env" files configuring the application.');
+} else {
+    // load all the .env files
+    (new Dotenv(true))->loadEnv(dirname(__DIR__) . '/.env');
+}
+
+$_SERVER['APP_ENV'] = $_ENV['APP_ENV'] = ($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null) ?: 'dev';
```
Comment (pamil) [inline src/Bundle/test/config/bootstrap.php:30]: To be consistent with the old `app/console` behaviour:

```suggestion
$_SERVER['APP_ENV'] = $_ENV['APP_ENV'] = ($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null) ?: 'test';
```

**Review by pamil — COMMENTED** [review]
Cool, thanks! Just one comment to make sure we don't break anything unintentionally.

**Comment by loic425** [comment]
> Cool, thanks! Just one comment to make sure we don't break anything unintentionally.

@pamil Ok now :)

**Comment by pamil** [comment]
Thank you, Loïc! :tada:

**Pull request merged**